advisories/github-reviewed/2023/08/GHSA-fh2r-99q2-6mmg/GHSA-fh2r-99q2-6mmg.json

```diff
@@ -20,11 +20,6 @@
         "ecosystem": "crates.io",
         "name": "rustls-webpki"
       },
-      "ecosystem_specific": {
-        "affected_functions": [
-          ""
-        ]
-      },
```

Comment:
I did not remove this; at least not intentionally.

Is it fine if this is removed?

Reply:
It's all good. Known bug and it'll get regenerated. Many thanks for the PR :)